python/cog/__init__.py

```diff
@@ -36,4 +37,5 @@
     "Input",
     "Path",
     "Secret",
+    "CommonChatSchemaChatMessage",
```

Comment:
Please use alphanumeric ordering :)

Reply:
Resolved in [ec7ac31](https://github.com/replicate/cog/pull/2186/commits/ec7ac313c0dbe7466c3973a584f8d4c9f067b83f)